TUTORIAL.md

```diff
@@ -261,7 +262,8 @@ We can now open the CSV file with any spreadsheet program or code library suppor
 
 ```python
 from datapackage import Package
-pkg = Package('country_population/datapackage.json')
+
```

Comment:
PEP-8 recommends separating import statements from the rest of the code with two blank lines.